containers/base/javascript/browser/css/dashboard.css

```diff
@@ -47,6 +83,93 @@ progress[value]::-moz-progress-bar {
   background-color: #391;
 }
 
+pre {
+    font-family: 'Inconsolata', monospace;
+    font-size: 1.1em;
+}
+
+a{
+    color: var(--premium-blue) !important;
+}
+
+.active-interface{
+    margin: var(--spacing-large);
+}
+
+.fbd-header .mdl-menu__container {
+  z-index: 50;
+  margin: 0 !important;
+  background-color: white;
+}
+
+.mdl-textfield--expandable {
+  width: auto;
+}
+
+.fbd-navigation-row{
+  display: flex;
+  background-color: white;
+}
+
+.fbd-title{
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.fbd-logo-image {
+  height: 80%;
+}
+
+.fbd-header {
+  overflow: visible;
+  background-color: white;
+}
+
+.fbd-header .mdl-navigation__link{
+  color: var(--premium-black) !important;
+  font-weight: var(--medium-font-weight);
+}
+
+.github_username{
+  color: var(--premium-black);
+  font-weight: 700;
+  font-size: 14px;
+  padding-right: 1em;
+}
+
+.fbd-navigation-container{
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.fbd-navigation .mdl-navigation__link {
+  display: inline-block;
+  height: 60px;
+  line-height: 68px;
```

Comment:
Looks like you'er trying to center content? Maybe `flex`? Otherwise see if you can lose the magic numbers